rest/model/project.go

```diff
@@ -373,6 +373,33 @@ type APIWorkstationConfig struct {
 	GitClone      *bool                        `bson:"git_clone" json:"git_clone"`
 }
 
+type APIContainerResources struct {
+	MemoryMB *int `bson:"memory_mb" json:"memory_mb"`
+	CPU      *int `bson:"cpu" json:"cpu"`
+}
+
+func (cr *APIContainerResources) BuildFromService(h interface{}) error {
+	var def model.ContainerResources
+	switch h.(type) {
+	case model.ContainerResources:
+		def = h.(model.ContainerResources)
+	case *model.ContainerResources:
+		def = *h.(*model.ContainerResources)
+	default:
+		return errors.Errorf("Invalid container resources of type '%T'", h)
+	}
+	cr.MemoryMB = utility.ToIntPtr(def.MemoryMB)
+	cr.CPU = utility.ToIntPtr(def.CPU)
+	return nil
+}
+
+func (cr *APIContainerResources) ToService() (interface{}, error) {
```

Comment:
Same thing as above, you don't need to return an `interface{}` if you prefer to just pass back `model.ContainerResource`, since this isn't an admin setting or an event log.

Reply:
Sure sounds much cleaner to me